portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -99,4 +106,15 @@ private static void storeComments(String name, String email, String comment, lon
     taskEntity.setProperty(ENTITY_TIMESTAMP_HEADER, timestamp);
     datastore.put(taskEntity);
   }
+
+  private static String translateComment(String originalText, String languageCode) {
```

Comment:
Ditto. Use sourceText and targetLanguageCode.

Reply:
Done